packages/core/src/models/CustomType/index.ts

```diff
@@ -16,7 +16,6 @@ export const CustomTypeSM = t.exact(
     }),
     t.partial({
       hash: t.string,
```

Comment:
what's hash for?

Reply:
I think it's a property added by the custom type API.

Reply:
I don't see any real use beside that